markdownify/__init__.py

```diff
@@ -398,18 +398,19 @@ def convert_p(self, el, text, convert_as_inline):
             # Preserve newlines (and preceding whitespace) resulting
             # from <br> tags.  Newlines in the input have already been
             # replaced by spaces.
-            lines = text.split('\n')
-            new_lines = []
-            for line in lines:
-                line = line.lstrip()
-                line_no_trailing = line.rstrip()
-                trailing = line[len(line_no_trailing):]
-                line = fill(line,
-                            width=self.options['wrap_width'],
-                            break_long_words=False,
-                            break_on_hyphens=False)
-                new_lines.append(line + trailing)
-            text = '\n'.join(new_lines)
+            if self.options['wrap_width'] is not None:
```

Comment:
Would it make sense to merge this with the preceding `if`? Disregard this comment if it does not.

Reply:
@AlexVonB - sharp eyes! Originally I had it on the higher-level `if`, but I decided to move it down here to reinforce the command about newlines being replaced by spaces when `wrap==True`. It added a level of indent and made the diff bigger, but runtime is practically identical and I felt the code was a bit clearer this way.